svg.go

```diff
@@ -344,91 +399,64 @@ func (svg *svgParser) skipToEndTag(l *xml.Lexer, tt xml.TokenType) xml.TokenType
 	return tt
 }
 
-func (svg *svgParser) parseDefs(l *xml.Lexer) {
```

Comment:
This section was causing the elemStack to get corrupted. I have styles on one of my svg files that make reference to the top svg element and they would never accept the elements because that element was missing from the stack.